leo_fw/src/firmware_message_converter.cpp

```diff
@@ -28,6 +29,17 @@ static ros::Subscriber imu_sub;
 static ros::Publisher imu_pub;
 static bool imu_advertised = false;
 
+static ros::Publisher odom_merged_pub;
+static ros::Timer odom_merged_timer;
+static geometry_msgs::Point odom_merged_position;
+static float odom_merged_yaw;
+static bool odom_merged_advertised = false;
+static float velocity_linear_x = 0;
+static float velocity_angular_z = 0;
+ros::ServiceClient odom_reset_client;
+ros::ServiceServer reset_odometry_service;
+static constexpr float PI = 3.141592653F;
```

Comment:
Put this constant at the top after includes, remove static as its not needed and change float to double.